common/src/main/java/com/wynntils/core/webapi/WebManager.java

```diff
@@ -28,45 +26,37 @@
 
     public static @Nullable WebReader apiUrls = null;
 
-    private static final SingleStaticProvider<HashMap<String, ItemGuessProfile>>
-            itemGuessesProvider;
+    private static final HashMap<String, ItemGuessProfile> itemGuesses = new HashMap<>();
 
     public static void init() {
         tryReloadApiUrls(false);
     }
 
-    static {
-        itemGuessesProvider =
-                new SingleStaticProvider<>() {
-                    @Override
-                    protected Request getRequest() {
-                        return new RequestBuilder(apiUrls.get("ItemGuesses"), "item_guesses")
-                                .cacheTo(new File(API_CACHE_ROOT, "item_guesses.json"))
-                                .handleJsonObject(
-                                        json -> {
-                                            Type type =
-                                                    new TypeToken<
-                                                            HashMap<
-                                                                    String,
-                                                                    ItemGuessProfile>>() {}.getType();
-
-                                            GsonBuilder gsonBuilder = new GsonBuilder();
-                                            gsonBuilder.registerTypeHierarchyAdapter(
-                                                    HashMap.class,
-                                                    new ItemGuessProfile.ItemGuessDeserializer());
-                                            Gson gson = gsonBuilder.create();
-
-                                            value = gson.fromJson(json, type);
-                                            return true;
-                                        })
-                                .useCacheAsBackup()
-                                .onError(
-                                        () -> {
-                                            value = new HashMap<>();
-                                        })
-                                .build();
-                    }
-                };
+    private static void loadItemGuesses() {
+        handler.addRequest(
+                new RequestBuilder(apiUrls.get("ItemGuesses"), "item_guesses")
+                        .cacheTo(new File(API_CACHE_ROOT, "item_guesses.json"))
+                        .handleJsonObject(
+                                json -> {
+                                    Type type =
```

Comment:
spotless is really having a weird opinion on how good code should look. Maybe we should go back to the drawing board and start looking for another formatter. This looks horrible and almost unreadable, imho.

Reply:
Yeah, lambdas/builders aren’t handled well